src/Google.Api.Gax/Api/Gax/GaxPreconditions.cs

```diff
@@ -189,5 +190,12 @@ public static void CheckArgument<T>(bool condition, string paramName, string for
                 throw new ArgumentException(string.Format(format, arg0, arg1), paramName);
             }
         }
+
+        internal static void CheckNotNull<TRequest, TResponse>(ApiCallable<TRequest, TResponse> originalCall)
+            where TRequest : class, IMessage<TRequest>
+            where TResponse : class, IMessage<TResponse>
+        {
+            throw new NotImplementedException();
```

Comment:
This is temporary, wondering if this should have also the name of the parameter?